src/ansys/fluent/core/solver/flobject.py

```diff
@@ -32,6 +32,12 @@
 settings_logger = logging.getLogger("pyfluent.settings_api")
 
 
+class InactiveObjectError(RuntimeError):
+    """Raises exception if the object is inactive."""
```

Comment:
This _is_ an exception class, so the docstring is inaccurate.

Reply:
Right, Thank you.

Reply:
Done. Thank you.